samcli/lib/package/artifact_exporter.py

```diff
@@ -144,6 +148,8 @@ def __init__(
             self.template_dir = template_dir
             self.code_signer = code_signer
         self.template_dict = yaml_parse(template_str)
+        if normalize_template:
+            ResourceMetadataNormalizer.normalize(self.template_dict)
```

Comment:
We are normalizing template dictionary here and above, is one of them redundant?

Reply:
resolved